api/controllers/projectNotification.js

```diff
@@ -0,0 +1,82 @@
+const defaultLog = require('winston').loggers.get('default');
+const mongoose = require('mongoose');
+const Actions = require('../helpers/actions');
+const Utils = require('../helpers/utils');
+const constants = require('../helpers/constants');
+
+exports.protectedOptions = function (args, res) {
+  res.status(200).send();
+};
+
+//  Create a new Project Notification
+exports.protectedPost = async function (args, res) {
+  const requestData = args.swagger.params.projectNotification.value;
+  defaultLog.info('Incoming new project notification object:', requestData);
+
+  const ProjectNotification = mongoose.model('ProjectNotification');
+  const projectNotification = new ProjectNotification({
+    ...requestData,
+    read: constants.SECURE_ROLES,
+    write: constants.SECURE_ROLES,
+    delete: constants.SECURE_ROLES
+  });
+
+  if (args.swagger.params.publish && args.swagger.params.publish.value) {
+    projectNotification.read.push('public');
+  }
+
+  try {
+    const saveProjectNotification = await projectNotification.save();
+    Utils.recordAction('Post', 'ProjectNotification', args.swagger.params.auth_payload.preferred_username, saveProjectNotification._id);
+    defaultLog.info('Saved new project notification object:', saveProjectNotification);
+    return Actions.sendResponse(res, 200, saveProjectNotification);
```

Comment:
Response code for a create should be a 201

Reply:
Updated